### PR TITLE
Update metadata API retention limits

### PR DESCRIPTION
**Pull request by runleonarun**
## Description & motivation

The metadata API can query the previous 90 days worth of data, so we need to capture this.
Notion request: https://www.notion.so/dbtlabs/Query-metadata-API-for-data-from-the-previous-3-months-25af72ad73f74f569628fc3fc62c7d35

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`
- [ ] [Run link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview.md [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
 
 The metadata API is available to accounts on the _Team_ and _Enterprise_ plans, for any version >= dbt v0.19.0. Please note that artifacts generated with a version of dbt _less than_ v0.19.0 will not be accessible via the dbt Cloud Metadata API.
 
+## How to browse the API
+
+We also provide a [graphical explorer](https://metadata.cloud.getdbt.com/graphiql) for this API where you can run ad-hoc queries or browse the schema. As GraphQL provides a self-describing API, the schema shown in the GraphiQL interface is an accurate representation of the graph and fields available to query on at any point in time.
+
+## Retention limits for data
+
+You can query data from the previous 90 days using the Metadata API.
```
Comment (ashnm) [inline website/docs/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview.md]: I think we should write 3 months, technically we're exposing slightly more than 90 days to ensure we cover a 3 month time line. We're using moment.js to include exactly three months, so if we to use today as an example the user could query June 27th through today.

Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview.md]:
```suggestion
You can use the Metadata API to query data from the previous 3 months. For example, If today is March 1, you can query data back to January 1st.
```

Comment (runleonarun) [reply website/docs/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview.md]: OK rewrote that, @ashnm!

File: website/docs/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview.md [hunk 2]
```diff
@@ -1,12 +1,22 @@
...
+The metadata API is available to accounts on the _Team_ and _Enterprise_ plans, for any version >= dbt v0.19.0. Please note that artifacts generated with a version of dbt _less than_ v0.19.0 will not be accessible via the dbt Cloud metadata API.
+
+## How to browse the API
+
 We also provide a [graphical explorer](https://metadata.cloud.getdbt.com/graphiql) for this API where you can run ad-hoc queries or browse the schema. As GraphQL provides a self-describing API, the schema shown in the GraphiQL interface is an accurate representation of the graph and fields available to query on at any point in time.
 
-The metadata API is available to accounts on the _Team_ and _Enterprise_ plans, for any version >= dbt v0.19.0. Please note that artifacts generated with a version of dbt _less than_ v0.19.0 will not be accessible via the dbt Cloud Metadata API.
+## Retention limits for data
+
+You can use the metadata API to query data from the previous 3 months. For example, If today is March 1, you can query data back to January 1st.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview.md]:
```suggestion
You can use the metadata API to query data from the previous 3 months. For example, if today was March 1, you could query data back to January 1st.
```

File: website/docs/docs/dbt-cloud/release-notes/05-Sept-2002/metadata-api-data-retention-limits.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Query metadata API for data from the previous 3 months"
+id: "metadata-api-data-retention-limits.md"
+description: "Metadata API data retention limits"
+sidebar_label: "Fix: Metadata API data retention limits"
+tags: [Sept-29-2022]
+---
+
+In order to make the metadata API more scalable and improve its latency, we’ve implemented data retention limits. The metadata API can now query data from the previous 3 months. For example, If today is March 1, you can query data back to January 1st.
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/release-notes/05-Sept-2002/metadata-api-data-retention-limits.md]:
```suggestion
In order to make the metadata API more scalable and improve its latency, we’ve implemented data retention limits. The metadata API can now query data from the previous 3 months. For example, if today was March 1, you could query data back to January 1st.
```

File: website/docs/docs/dbt-cloud/release-notes/05-Sept-2002/metadata-api-data-retention-limits.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Query metadata API for data from the previous 3 months"
+id: "metadata-api-data-retention-limits.md"
+description: "Metadata API data retention limits"
+sidebar_label: "Fix: Metadata API data retention limits"
+tags: [Sept-29-2022]
+---
+
+In order to make the metadata API more scalable and improve its latency, we’ve implemented data retention limits. The metadata API can now query data from the previous 3 months. For example, if today was March 1, you could query data back to January 1st.
```
Comment (matthewshaver) [inline website/docs/docs/dbt-cloud/release-notes/05-Sept-2002/metadata-api-data-retention-limits.md]: I don't think we discussed this as part of the style guide (nor do I necessarily think it needs to be included) but do we want to standardize numbers - should it be '3' or 'three' if it's not something like a date or a phone number?

File: website/docs/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview.md [hunk 5]
```diff
@@ -1,12 +1,22 @@
...
 id: "metadata-overview"
 ---
 
+## About the metadata API
+
 Every time that dbt Cloud runs a dbt project, it generates metadata which pertains to the accuracy, recency, configuration, and structure of the <Term id="view">views</Term> and tables in the warehouse. dbt Cloud serves a GraphQL API which supports arbitrary queries over this metadata; the endpoint for this API is `https://metadata.cloud.getdbt.com/graphql`. This API is an incredibly rich resource for evaluating data health longitudinally or at a point in time.
 
+## Prerequisites
+
+The metadata API is available to accounts on the _Team_ and _Enterprise_ plans, for any version >= dbt v0.19.0. Please note that artifacts generated with a version of dbt _less than_ v0.19.0 will not be accessible via the dbt Cloud metadata API.
```
Comment (matthewshaver) [inline website/docs/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview.md]: Is there a page we can link to that discusses how to upgrade a plan if they don't meet the pre-reqs?

**Review by matthewshaver — APPROVED** [review]

File: website/docs/docs/dbt-cloud/release-notes/05-Sept-2002/metadata-api-data-retention-limits.md (hunk 4 above)
Comment (runleonarun) [reply website/docs/docs/dbt-cloud/release-notes/05-Sept-2002/metadata-api-data-retention-limits.md]: Good call! I think I should write it out as "three" unless it's over 10 or whatever that ding dang rule is :)

File: website/docs/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview.md (hunk 5 above)
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview.md]:
```suggestion
The metadata API is available to accounts on the _Team_ and _Enterprise_ plans, for any version >= dbt v0.19.0. Please note that artifacts generated with a version of dbt _less than_ v0.19.0 will not be accessible via the dbt Cloud metadata API. For information on upgrading, see "[Version migration guides](https://docs.getdbt.com/guides/migration/versions)."
```

File: website/docs/docs/dbt-cloud/release-notes/05-Sept-2002/metadata-api-data-retention-limits.md (hunk 4 above)
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/release-notes/05-Sept-2002/metadata-api-data-retention-limits.md]:
```suggestion
In order to make the metadata API more scalable and improve its latency, we’ve implemented data retention limits. The metadata API can now query data from the previous three months. For example, if today was March 1, you could query data back to January 1st.
```

File: website/docs/docs/dbt-cloud/release-notes/05-Sept-2002/metadata-api-data-retention-limits.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Query metadata API for data from the previous 3 months"
```
Comment (runleonarun) [inline website/docs/docs/dbt-cloud/release-notes/05-Sept-2002/metadata-api-data-retention-limits.md]:
```suggestion
title: "Query metadata API the previous three months of data"
```

File: website/docs/docs/dbt-cloud/release-notes/05-Sept-2002/metadata-api-data-retention-limits.md (hunk 4 above)
Comment (runleonarun) [reply website/docs/docs/dbt-cloud/release-notes/05-Sept-2002/metadata-api-data-retention-limits.md]: I made this change and we can follow up with the team to come together with standards

File: website/docs/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview.md (hunk 5 above)
Comment (runleonarun) [reply website/docs/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview.md]: Added a link to the upgrade guides!

**Comment by runleonarun** [comment]
@ashnm this is ready to merge in the morning when you give the word (or you can click **Merge** if you'd like!)

**Pull request merged**